cf/tasks/package-bits.sh

```diff
@@ -30,8 +30,9 @@ do
     mkdir tmp
 
     # kube configs
-    # helm charts
     unzip s3.kube-dist/scf-kube-*.zip -d tmp
+    # helm charts
+    unzip s3.helm-dist/scf-helm-*.zip -d tmp
```

Comment:
Seems that you are correct. The helm stuff is under `cf` as well, so it mixes with the kube cf.
Now fixing.